src/draw/lv_draw.c

```diff
@@ -3,6 +3,10 @@
  *
  */
 
+/**
+ * Modified by NXP in 2024
+ */
```

Comment:
In LVGL I see a lot of comments style.
```
/**
 * ... text ...
 */
```
or
```
/*
 ... text ...
*/
```
or
```
/*********************************************
 *  ... text
 *********************************************/
```
It was no problem so far. But hence, what is the accepted comment style? All of them?

Reply:
> @vwheeler63 will the `/**` trigger unwanted Doxygen behavior?

Hi, Liam!

Technically no, but what it _is_ causing is probably not intentional:  it causes Doxygen to parse that comment, but then Doxygen will look for the code member it should assign that documentation to, and it won't find anything, so it will be ignored.  So it should either be:

1. If Gabor is okay with that comment being part of the FILE's documentation, then I would move it into the top comment after the `@file` command.
2. Otherwise, it should only have one asterisk:  `/*`, even though it won't hurt anything as explained.

Reply:
> Otherwise, it should only have one asterisk: `/*`, even though it won't hurt anything as explained.

That's what I was wondering just because of [this thread](https://github.com/lvgl/lvgl/pull/6584#discussion_r1706008049). Glad it's harmless regardless.

Reply:
> Glad it's harmless regardless.

**thumbs up in agreement**